docs/api/security/psa_attestation.md

```diff
@@ -0,0 +1,19 @@
+## PSA Attestation
+
+### Description
+
+TODO: AlexV
```

Comment:
@avolinski  - please handle

Reply:
@moranpeker provided this in a separate PR (https://github.com/ARMmbed/mbed-os-5-docs/pull/967).

Reply:
do you want me to revert this file - please advise?

Reply:
@moranpeker updated the file.